### PR TITLE
Fixes #10: Migrate MIDI Port to Atom

### LINKED ISSUE #10: MIDI input port *MUST* be migrated to Atom
Hi @nicoroulet !

We have been using the Raffo emulator with zynthian for a while, but suddenly, after a system update, it doesn't work any more. 

The problem is that the LV2 Event extension is totally deprecated and MIDI input port should be migrated to Atom.

I've been trying a little bit, but only got segfaults until now ....

### PR DESCRIPTION
**Pull request by jofemodo**
Also improves Makefile, allowing easy external setting of optimization flags.

### CODE REVIEW

File: raffo.cpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ RaffoSynth::RaffoSynth(double rate):
 , output("data/oscasm_256.out")
 #endif
   {
-    midi_type = Parent::uri_to_id(LV2_EVENT_URI, "http://lv2plug.in/ns/ext/midi#MidiEvent"); 
+      //midi_type = Parent::uri_to_id(LV2_EVENT_URI, "http://lv2plug.in/ns/ext/midi#MidiEvent");
```
Comment (nicoroulet) [inline raffo.cpp]: Please remove this line altogether

File: raffo.cpp [hunk 2]
```diff
@@ -250,9 +252,9 @@ void RaffoSynth::run(uint32_t sample_count) {
       samples_done = to;
     }
 
-    if (ev) {
-      if (ev->type == midi_type)
-        static_cast<RaffoSynth*>(this)->handle_midi(ev->size, event_data);
+    if (iter) {
+      //if (type == midi_type)
```
Comment (nicoroulet) [inline raffo.cpp]: remove this line too and fix indentation of line below

**Review by nicoroulet — APPROVED** [review]

File: raffo.cpp (hunk 1 above)
Comment (jofemodo) [reply raffo.cpp]: Done!

File: raffo.cpp (hunk 2 above)
Comment (jofemodo) [reply raffo.cpp]: Done!

**Review by jofemodo — COMMENTED** [review]
I think it's ready to merge ;-)

**Pull request merged**